packages/bitstringStatusList/package.json

```diff
@@ -42,6 +42,7 @@
     "@transmute/did-web": "0.7.0-unstable.82",
     "@transmute/vc-status-rl-2020": "0.7.0-unstable.82",
     "@vckit/core-types": "workspace:*",
+    "@vckit/credential-jwt-enveloping-proof": "workspace:1.0.0-beta.7",
```

Comment:
Does bitstringStatus need it?

Reply:
Yes, we need the suite to get JsonWebKey2020 to issue/verify Bit String